### ISSUE TITLE
Has the behavior of "Y" changed from 0.5 to 0.6?

### ISSUE THREAD

**Issue by d-mceneaney** [neovim/neovim#15426]
OS: Windows 10
Neovim Version: 0.6

Observation: When the cursor is in the middle of a line, selecting "Y" in normal mode yanks from the current cursor position to the end of the line (behaving like "C"  and "D").

In Neovim version 0.5, selecting "Y" in normal mode yanks the entire line, regardless of cursor position. 

Is this change intentional for 0.6 or is it an issue?

Thanks.

**Comment by clason** [comment]
Yes: https://github.com/neovim/neovim/pull/13268

**Comment by justinmk** [comment]
Noted in https://github.com/neovim/neovim/issues/14090#issuecomment-901882455

**Comment by d-mceneaney** [comment]
Thanks folks.

**Comment by shibotto** [comment]
How can I restore the original behavior?

**Comment by clason** [comment]
`unmap Y`

**Comment by jamessan** [comment]
> How can I restore the original behavior?

`:help default-mappings`

**Comment by pulzar** [comment]
> Did you put anything _after_ `unmap Y`? Even spaces are bad.

I think that must've been it. I "fixed" it by moving it to the beginning of the file, but after your comment I went back and tried it at the end again, and it worked fine... then I added a space after Y, and it failed with the original error. So, I must've had a space or something after the command which I cleaned up when I moved it. 

Either way - thank you, I learned something new :)... it was user error, after all.
